src/cache.jl

```diff
@@ -113,7 +113,7 @@ function updatecache(absentmodules::Vector{Symbol}, server)
     
     @async begin 
         str = readline(o)
-        data = base64decode(str)
+        data = base64decode(chomp(str))
```

Comment:
I think that is actually a bug on julia ``master``, see JuliaLang/julia#21314. Maybe we'll wait and see what they say over there? If they fix that quickly, we can wait here with the merge, otherwise we can go with the copy approach?